src/SERVER/src/args/parse_args/c_per_teams.c

```diff
@@ -0,0 +1,31 @@
+/*
+** EPITECH PROJECT, 2023
+** zappy server args parse
+** File description:
+** height
+*/
+
+#include <stdbool.h>
+#include "tlcstdlibs.h"
+#include "tlcstrings.h"
+#include "args.h"
+#include "internal.h"
+
+bool parse_arg_c_per_teams(const char *const arr[], args_t *args)
+{
+    if (arr == NULL || arr[0] == NULL || x_strcmp(arr[0], "-c") != 0 ||
```

Comment:
magic value, already exist in a static array in args_init.c, same error with 32 others "-c" and with others elements of the static array